examples/StandardFirmataBLE/bleConfig.h

```diff
@@ -106,6 +106,15 @@ BLEStream stream;
 // BLEStream stream(BLE_REQ, BLE_RDY, BLE_RST);
 // #endif
 
+/*
+ * Adafruit Bluefruit nRF52 Boards
+ */
+#ifdef ARDUINO_NRF52_ADAFRUIT
+#include "utility/BluefruitStream.h"
+BLEStream stream;
+#endif
```

Comment:
ah, I kind of guess what you mean. Though the wiring_analog.c does not include the bleconfig.h there is no way for it to know if the macro existed.